menas/ui/components/dataset/addDataset.fragment.xml

```diff
@@ -29,20 +29,37 @@
                         <core:Fragment type="XML" fragmentName="components.schema.selector.schemaSelector"/>
                         <Label text="HDFS raw data folder path"/>
                         <VBox>
-                        	<Input type="Text" id="selectedRawHDFSPathLabel"/>
-                            <hdfs:HDFSBrowser id="newDatasetRawHDFSBrowser" height="300px" width="100%"
+                            <Input type="Text" id="selectedRawHDFSPathLabel"/>
+                            <hdfs:HDFSBrowser id="newDatasetRawHDFSBrowser" height="250px" width="100%"
                                           busyControl="addDatasetDialog"
                                           pathInput ="selectedRawHDFSPathLabel" restURI="api/hdfs/list"
                                           HDFSPath="{entity>/hdfsPath}"/>
                         </VBox>
                         <Label text="HDFS conformed data publish folder path"/>
                         <VBox>
                             <Input type="Text" id="selectedPublishHDFSPathLabel"/>
-                            <hdfs:HDFSBrowser id="newDatasetPublishHDFSBrowser" height="300px" width="100%"
+                            <hdfs:HDFSBrowser id="newDatasetPublishHDFSBrowser" height="250px" width="100%"
                                           busyControl="addDatasetDialog"
                                           pathInput="selectedPublishHDFSPathLabel" restURI="api/hdfs/list"
                                           HDFSPath="{entity>/hdfsPublishPath}"/>
                         </VBox>
+                        <Label text="Dataset Tags"/>
+                        <List width="100%" items="{entity>/_properties}">
+                          <InputListItem class="inputListItemSmallLabel" label="{entity>name}" tooltip="{entity>description}">
+                            <HBox alignItems="Center" justifyContent="End">
+                                <Input type="Text" width="20rem" visible="{= ${entity>propertyType/_t} === 'StringPropertyType'}" 
+                                    value="{entity>value}" valueState="{entity>validation}" valueStateText="{entity>validationText}"></Input>
+                                <Select width="20rem" items="{path: 'entity>propertyType/allowedValues', templateShareable: false}" 
+                                    selectedKey="{entity>value}" valueState="{entity>validation}" valueStateText="{entity>validationText}"
+                                    visible="{= ${entity>propertyType/_t} === 'EnumPropertyType'}">
+                                    <core:Item key="{entity>value}" text="{entity>value}"></core:Item>
+                                </Select>
+                                <ObjectStatus text="{entity>essentiality/_t}" class="essentialityStatus" 
```

Comment:
Is it possible to have the icon on the right?
![image](https://user-images.githubusercontent.com/17467768/106569716-9446a400-653d-11eb-8596-285734ef4e44.png)

Reply:
Yeah it is, it didn't look very good :) can try style it up a bit more